MSAL/test/integration/native_auth/end_to_end/otp_code_retriever/MSALNativeAuthEmailCodeRetriever.swift

```diff
@@ -29,6 +29,14 @@ class MSALNativeAuthEmailCodeRetriever: XCTestCase {
     private let baseURLString = "https://www.1secmail.com/api/v1/?action="
     private let secondsToWait = 4.0
     private let numberOfRetry = 3
+    private let dateFormatter: DateFormatter = {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "yyyy-MM-dd HH:mm:ss"
+        dateFormatter.locale = Locale(identifier: "en_US_POSIX")
+        dateFormatter.timeZone = TimeZone(secondsFromGMT: 0) //We ignore the timezone
+        return dateFormatter
+    }()
+    private let secondsToWaitForEmail = 5.0
```

Comment:
What is the difference between "secondsToWait" on line 30 and this attribute "secondsToWaitForEmail". Do we need both? Using the name, I can't understand the difference between the two

Reply:
Renamed variable to be more clear. It's actually the maximum number of seconds that should have passed since the email has been receibed.